test/unittests.html

```diff
@@ -16,7 +16,7 @@
     <script src="lib/fetch.js"></script>
     <script>
         mocha.setup('bdd');
-        mocha.timeout(240000);
+        mocha.timeout(300000);
```

Comment:
These times are the limit for individual test cases. Not total build time. I am aware that the cross browser build on saucelabs is currently not very stable, as some browsers timetimes time out (probably due to memory issues). Increasing this value will only reduce our ability to catch performance regressions :/